pkgs/by-name/qp/qpoases/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qpoases";
+  version = "3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "coin-or";
+    repo = finalAttrs.pname;
+    rev = "releases/${finalAttrs.version}";
+    hash = "sha256-NWKwKYdXJD8lGorhTFWJmYeIhSCO00GHiYx+zHEJk0M=";
+  };
+
+  patches = [
+    # Allow building as shared library.
+    # This was merged upstream, and can be removed on next version
```

Comment:
Can you please add a link to upstream PR or commit ?

Reply:
Well, this is already in the url. Should I duplicate this info ?

Reply:
Fair enough.